examples/jwt-validator/methods/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.1.0"
 edition = "2021"
 
 [build-dependencies]
-risc0-build = { path = "../../../risc0/build" }
+risc0-build = { path = "../../../risc0/build", features = ["unstable"] }
```

Comment:
Won't this cause the entire workspace to allow unstable to be visible?

Reply:
Hmmm, quite possibly yes. At least in the build script, so the `RISC0_FEATURE_bigint2` env var will be set. This won't make the `sys_bigint2` symbols available though. This is an interesting result of a common workspace.